iphone/Classes/TiUILabel.m

```diff
@@ -225,96 +225,77 @@ -(void)handleListenerRemovedWithEvent:(NSString *)event
     }
 }
 
--(BOOL)checkLinkAttributeForString:(NSMutableAttributedString*)theString atPoint:(CGPoint)p
+- (NSTextContainer*)currentTextContainer
 {
-    CGPoint thePoint = [self convertPoint:p toView:label];
-    CGRect drawRect = [label textRectForBounds:[label bounds] limitedToNumberOfLines:label.numberOfLines];
-    drawRect.origin.y = (label.bounds.size.height - drawRect.size.height)/2;
-    thePoint = CGPointMake(thePoint.x - drawRect.origin.x, thePoint.y - drawRect.origin.y);
-    //Convert to CT point;
-    thePoint.y = (drawRect.size.height - thePoint.y);
-    CTFramesetterRef theRef = CTFramesetterCreateWithAttributedString((__bridge CFAttributedStringRef)theString);
-    if (theRef == NULL) {
-        return;
-    }
+    NSTextContainer *textContainer = [[NSTextContainer alloc] initWithSize:self.label.bounds.size];
+    textContainer.lineFragmentPadding  = 0;
+    textContainer.maximumNumberOfLines = (NSUInteger)self.label.numberOfLines;
+    textContainer.lineBreakMode = self.label.lineBreakMode;
+    return textContainer;
```

Comment:
Xcode Analyzer, great tool to find memory leaks in your code without running the app:
![ev3wb](https://cloud.githubusercontent.com/assets/10667698/19975324/d3e071e6-a1eb-11e6-80ca-8bdf663af5a5.png)